### ISSUE TITLE
Unavailable repository 'calculate' referenced by masters entry

### ISSUE THREAD

**Issue by Massimo-B** [rasdark/overlay#5]
Hi, I'm on Gentoo and always get the annoying warning from Portage:

Unavailable repository 'calculate' referenced by masters entry in '/var/lib/layman/rasdark/metadata/layout.conf'

because of
https://github.com/rasdark/overlay/blob/a4f5aca4170d5341d6d98e0c1818735d878ca79b/metadata/layout.conf#L1

**Comment by nikmoon** [comment]
layman -a calculate

**Comment by p4r4D0xum** [comment]
layman is deprecated, please use `eselect repository enable <repo>`.

**Comment by vitaly-zdanevich** [comment]
After `eselect repository enable calculate` I got:

```
!!! Section 'calculate' in repos.conf has location attribute set to nonexistent directory: '/var/db/repos/calculate'
Unavailable repository 'calculate' referenced by masters entry in '/var/db/repos/rasdark/metadata/layout.conf'
WARNING: One or more repositories have missing repo_name entries:

	/var/db/repos/calculate/profiles/repo_name

NOTE: Each repo_name entry should be a plain text file containing a
unique name for the repository on the first line.
```

**Comment by rasdark** [comment]
fixed in commit ceadb2e
